entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash +x
+
+/opt/orbs/orbs-node $@ | multilog t s16777215 n3 '!tai64nlocal' /opt/orbs/logs 2>&1
```

Comment:
Please document what these parameters mean

Reply:
nice to have - allow override 16777215 and 3 using env vars?